### PR TITLE
Add hint about upgrading Katello

### PR DESCRIPTION
**Pull request by maximiliankolb**
Refs https://community.theforeman.org/t/yet-another-foreman-upgrade-fail/38868/6?u=maximilian cc @jeremylenz 

* [x] I am okay with my commits getting squashed when you merge this PR.
* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.11/Katello 4.13
* [x] Foreman 3.10/Katello 4.12

### CODE REVIEW

**Review by jeremylenz — APPROVED** [review]
LGTM, thanks!

File: guides/doc-Upgrading_Project/master.adoc [hunk 1]
```diff
@@ -17,6 +17,13 @@ ifdef::satellite[]
 include::common/modules/proc_providing-feedback-on-red-hat-documentation.adoc[leveloffset=+1]
 endif::[]
 
+ifdef::foreman-el[]
+[NOTE]
```
Comment (apinnick) [inline guides/doc-Upgrading_Project/master.adoc:21]:
```suggestion
[IMPORTANT]
```

File: guides/doc-Upgrading_Project/master.adoc [hunk 2]
```diff
@@ -17,6 +17,13 @@ ifdef::satellite[]
...
+====
+If you use the Katello plugin, please use {BaseURL}Upgrading_Project/index-katello.html[Upgrading Foreman with Katello].
```
Comment (apinnick) [inline guides/doc-Upgrading_Project/master.adoc:23]:
```suggestion
If you have installed the Katello plugin, you must use {BaseURL}Upgrading_Project/index-katello.html[Upgrading Foreman with Katello].
```
I think this should be a bit more forceful.

**Review by apinnick — CHANGES_REQUESTED** [review]
I recommend a somewhat more direct tone. :-)

File: guides/doc-Upgrading_Project/master.adoc (hunk 1 above)
Comment (maximiliankolb) [reply guides/doc-Upgrading_Project/master.adoc]: Applied.

File: guides/doc-Upgrading_Project/master.adoc (hunk 2 above)
Comment (maximiliankolb) [reply guides/doc-Upgrading_Project/master.adoc]: Applied.

**Review by maximiliankolb — COMMENTED** [review]
Applied both suggestions.

**Review by apinnick — APPROVED** [review]

**Comment by apinnick** [comment]
Merged to 'master' and cherry-picked:

c59386bec4..06c3175d7c  3.11 -> 3.11
7e66719dc8..9ae458544a  3.10 -> 3.10

**Pull request merged**